FindMe/View Controllers/ProfileViewController.m

```diff
@@ -7,6 +7,10 @@
 //
 
 #import "ProfileViewController.h"
+#import "LoginViewController.h"
+#import "SceneDelegate.h"
+#import "Parse/Parse.h"
```

Comment:
This is undoubtedly wrong. VCs should never need to import other VCs. Also VCs should not know about parse.